ibis/backends/tests/test_client.py

```diff
@@ -1144,17 +1246,33 @@ def test_set_backend_url(url, monkeypatch):
 )
 @pytest.mark.broken(["oracle"], reason="oracle doesn't like `DESCRIBE` from sqlalchemy")
 @pytest.mark.broken(["druid"], reason="sqlalchemy dialect is broken")
+@pytest.mark.notimpl(
+    ["flink"],
+    raises=AttributeError,
+    reason="'Backend' object has no attribute 'raw_sql'",
+)
 def test_create_table_timestamp(con, temp_table):
     schema = ibis.schema(
         dict(zip(string.ascii_letters, map("timestamp({:d})".format, range(10))))
     )
-    con.create_table(temp_table, schema=schema, overwrite=True)
+
+    if con.name == "flink":
+        con.create_table(
+            temp_table,
+            schema=schema,
+            tbl_properties={
+                "connector": None,
+            },
```

Comment:
@mfatihaktas same concern as below around divergent API just for Flink

Reply:
Referring to the [reply above](https://github.com/ibis-project/ibis/pull/6920#discussion_r1372456721).